### ISSUE TITLE
Mod Organizer2 Instant CTD

### ISSUE THREAD

**Issue by wmcook32** [TanninOne/modorganizer#359]
When loading FO4 through MO the game starts loading but the screen is black and once you try to do the press any key to continue it instantly crashes.

**Comment by TanninOne** [comment]
please attach the logs/usvfs-<date>.log

**Comment by wmcook32** [comment]
[usvfs-2015-11-23_22-38-22.log.txt](https://github.com/TanninOne/modorganizer/files/42195/usvfs-2015-11-23_22-38-22.log.txt)

**Comment by Newyears1978** [comment]
I have the same issue here, vanilla game. Figured I would post it here rather than making another thread.
[usvfs-2015-11-25_06-46-12.txt](https://github.com/TanninOne/modorganizer/files/43541/usvfs-2015-11-25_06-46-12.txt)

**Comment by wmcook32** [comment]
I think I half know what the issue is, but no idea how to go about fixing it. The issue is this:

made .bak versions of the ini files in the default locals ie My Games/Fallout4.
It created Fallout4Prefs.ini but not Fallout4.ini

I thought that the virtualization of MO was supposed to be that it makes and uses copies of those files in the profiles folders.

**Comment by Newyears1978** [comment]
@wmcook32 IF that's the case, did you try to manually copy Fallout4.ini into the profile and see if it runs then?

I am at work now so I cannot test. I hope to get this working soon as I have unready removed NMM and need to reinstall my mods before I can play again ;) (Did a bit of building with some mods..even though I know that's unadvised..have not had any CTD or bugs because of it in 150 hours)

**Comment by wmcook32** [comment]
the profiles have the ini's but MO isn't reading them. It's reading the ones in My Docs>My Games>Fallout4. If you rename or remove the ini files and try to run FO4 through MO it only creates the prefs ini.

**Comment by Newyears1978** [comment]
Oh, I hope for a fix soon as I am not about to reinstall my mods with NMM and won't play until I can install them (or my base building will basically get destroyed)

**Comment by Newyears1978** [comment]
So I have determined as @wmcook32 said it is INDEED a issue where the directory is not being seen properly.

When it tries to fire up the movies it crashes because it cannot find them.

I copied my FO4 data directory with default Strings files and movies to a fake mod in my mod folder, and now MO will run.

Hopefully @TanninOne will see this and fix the issue

**Comment by wmcook32** [comment]
I just wanted to update my Issue. This morning I upgraded to windows 10 from windows 7. So far no crashes at start up like mentioned above. However the game is not recognizing the local saves option, but instead using the global saves. 

What this is telling me is that for some reason windows 7 is not accepting the proper calls to the proper files and or directories. And since the game atm is loading the global saves it leads me to believe that the program isn't using the virtual/profile options.

**Comment by Goofysoccer** [comment]
@wmcook32 Is there a way to fix this with Windows 7?

**Comment by wmcook32** [comment]
Atm I don't know. That would require TanninOne's input. I have a feeling it is internal linking not working. I can trouble shoot things but to actually fix it I can't since my programming is crude unless it is dealing with web programming.

**Comment by strangerism** [comment]
16:39:43.219 <%P:3776> [D] NtOpenFile [source=\??\C:\Users\Mark\Documents\My Games\Fallout4\Fallout4.INI] [rerouted=\??\E:/Program Files (x86)/Steam/steamapps/common/Fallout 4/ModOrganizer2/profiles/Joshua Anderson/fallout4.ini] [*FileHandle=<null>] [res=err c0000034]

I am guessing here, could you try to 1) move MO to another directory not under control of windows i.e. like C:\MO and 2) run MO as administrator after that (seems not necessary but give it a go if the first thing doesn't work)?

**Comment by wmcook32** [comment]
the issue is no longer happening since I upgraded to windows 10. The game begins but still doesn't actually follow certain rules/features MO has ie localsaves

**Comment by strangerism** [comment]
well localsaves is the less of the problems, I don't thing I have actually managed to run a plugin and some mods with just file resources don't even work.

**Comment by Newyears1978** [comment]
Yeah definitely looks like a problem with directories and virutalization..but then again some other people have reported it to work with Windows7 64bit.

I installed all my mods back with NMM as I wanted to play the game..I know MO will be ready some day.
